src/modules/RefreshmentMap/index.tsx

```diff
@@ -181,7 +181,7 @@ export const RefreshmentMap: FC<RefreshmentMapPropType> = (pageProps) => {
           <Sidebar {...pageProps} />
           <div
             className={classNames(
-              'absolute transform z-50',
+              'absolute transform z-50 pointer-events-none',
```

Comment:
This seems to make the `HourSelector` inaccessible by mouse click. Can you double-check that?

What is actually the reason to make the pointer-events `none`?

Reply:
This is on purpose as it prevents the scroll of sidebars and the panning of the map to happening. If you look at the interactive children of HourSelector, they set `pointer-events-auto` to counter this on a more local level.